src/terraform/index.go

```diff
@@ -12,7 +12,7 @@ import (
 
 // terraformModulesCommitHash is the git commit hash that reflects
 // which of the Terraform modules in Originate/exosphere we are using
-const terraformModulesCommitHash = "8786f912"
+const terraformModulesCommitHash = "48731e6"
```

Comment:
Hmm. Should this always be the latest commit (and thus we should try and automate this type of update) or one we want to update manually?

Reply:
We'll have to be updating this more often than during release if we want a working version of this command.

I think we'll always have to specify commit hashes, because as we release new versions of Exosphere we want to lock which Terraform modules are being used, instead of always pointing to the latest ones.